src/components/WidgetContainer/widget-container.jsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import { closeIcon, profile } from "../../assests/index";
+import WidgetButton from "../WidgetButton/widget-button";
+
+const WidgetContainer = () => {
+  const [isButtonClick, setIsButtonClick] = useState(false);
+  const [showIntroModal, setShowIntroModal] = useState(false);
+  const [timer, setTimer] = useState(null);
```

Comment:
Could you look into the framer-motion library and see if this might be an easy thing to animate using that library? It would be nice to have a simple "genie" effect, similar to when you minimize windows on a mac, if that's possible to do easily. If not, don't worry too much about it.


https://github.com/jahorwitz/edtonomy/assets/29721136/aa288c55-ba2a-4100-917d-b114923897ba